cnf-certification-test/preflight/suite.go

```diff
@@ -42,6 +44,13 @@ var _ = ginkgo.Describe(common.PreflightTestKey, func() {
 		return
 	}
 
+	// Safeguard against running the preflight tests if we are specifically targeting a certain label eg. in QE.
+	ginkgoConfig, _ := ginkgo.GinkgoConfiguration()
+	if !labelsAllowTestRun(ginkgoConfig.LabelFilter, []string{common.PreflightTestKey, identifiers.TagCommon}) {
+		logrus.Debug("LabelFilter is set but 'preflight' or 'common' tests are not targeted.  Issuing skip.")
```

Comment:
I'd rather use logrus.Warn or logrus.Info here. Also, there's an extra space before the word "Issuing".